index.html

```diff
@@ -61,10 +62,198 @@ <h3>LET’S CONNECT</h3>
+    <img src="images/Snapshoot2.png" alt="">
+    </div>
+    <div class="multipost">
+        <h2>Multi-Post
+            Stories</h2>
+    </div>
+    <div class="frame-1">
+<img src="images/canopy.png" alt="">
+    </div>
+    <div class="detail-1">
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    </div>
+    <div class="work-language-1">
+<ul>
+    <li>
+        <a href="#"><img src="images/Tag.png" alt=""></a>
+    </li>
+    <li>
+        <a href="#"><img src="images/Tag1.png" alt=""></a>
+    </li>
+    <li>
+        <a href="#"><img src="images/Tag2.png" alt=""></a>
+    </li>
+</ul>
+
+</div>
+    <button type="button" class="button-2">See Project</button>
+
+ </section>
+    <!-- card3 -->
+             
+<section class="work-container2">
+    <div class="img-snapshot3">
+    <img src="images/Snapshoot3.png" alt="">
+    </div>
+    <div class="tonic2">
+        <h2>Tonic</h2>
+    </div>
+    <div class="frame-2">
+<img src="images/canopy.png" alt="">
+    </div>
+    <div class="detail-2">
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    </div>
+    <div class="work-language-2">
+<ul>
+    <li>
+        <a href="#"><img src="images/Tag.png" alt=""></a>
+    </li>
+    <li>
+        <a href="#"><img src="images/Tag1.png" alt=""></a>
+    </li>
+    <li>
+        <a href="#"><img src="images/Tag2.png" alt=""></a>
+    </li>
+</ul>
+</div>
+  
+    <button type="button" class="button-3">See Project</button>
+
+</section>
+    <!-- card4 -->
+             
+<section class="work-container3">
+    <div class="img-snapshot4">
+    <img src="images/Snapshoot4.png" alt="">
+    </div>
+    <div class="multipost1">
+        <h2>Multi-Post
+            Stories</h2>
+    </div>
+    <div class="frame-3">
+<img src="images/canopy.png" alt="">
+    </div>
+    <div class="detail-3">
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    </div>
+    <div class="work-language-3">
+<ul>
+    <li>
+        <a href="#"><img src="images/Tag.png" alt=""></a>
+    </li>
+    <li>
+        <a href="#"><img src="images/Tag1.png" alt=""></a>
+    </li>
+    <li>
+        <a href="#"><img src="images/Tag2.png" alt=""></a>
+    </li>
+</ul>
+    </div>
+  
+    <button type="button" class="button-4">See Project</button>
+
+</section>
+
+
+    <section class="about-wrapper">
+        <div class="header">
+            <h2>About
+                Myself</h2>
+        </div>
+        <div class="about2">
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        </div>
+        <div class="connect">
+            <h3>LET’S CONNECT</h3>
+        </div>
+        <div class="socialmedia-icon">
```

Comment:
- Please try to stick as much as possible to the Figma design, which includes:
   - [ ] Adding the border-radius for this section (after adjusting the background of the works section), to match the Fimga template.
   - [ ] Using the same font style (font-weight and size) for the "About Myself" title and the "Get my resume" button, to match the Figma template.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/165793339-9cbd2062-db4b-44ca-8857-b642f1b3f6f0.png) | ![image](https://user-images.githubusercontent.com/62290545/165793400-bab073e9-1af0-4eb8-b73e-8f7fc0f45f1a.png) |